.github/workflows/zips.yml

```diff
@@ -113,59 +116,98 @@ jobs:
           name: zip-files
           path: ${{ github.ref }}
 
-  deploy-to-wiki:
-    name: Deploy To Wiki
+  upload-to-cloud-storage:
+    name: Upload to GCS
     runs-on: ubuntu-latest
     needs: build-zips
     steps:
-      - uses: actions/checkout@v3
-        with:
-          repository: ${{ github.repository }}.wiki
-          # GitHub Wikis use this term as their main branch; this cannot be
-          # changed so we need to continue using it for now.
-          ref: master
-      - name: Download artifacts
-        uses: actions/download-artifact@v3
+      - name: Set branch environment variables
+        id: branch-target
+        if: github.event_name != 'pull_request'
+        run: echo "dir=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
+      - name: Set PR environment variables
+        id: pr-target
+        if: github.event_name == 'pull_request'
+        run: echo "dir=pull/${{ github.event.number }}" >> $GITHUB_OUTPUT
+      - uses: actions/download-artifact@v3
         with:
           name: zip-files
           path: ${{ github.ref }}
-      - name: Commit updates
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+      - uses: google-github-actions/setup-gcloud@v1
+      - name: Upload PR files
+        run: gcloud storage cp --recursive ${{ github.ref }} gs://${{ env.BUCKET_NAME }}/builds/${TARGET_DIR}
```

Comment:
We can use the dedicated action for this which looks a bit cleaner for this action. See https://github.com/google/site-kit-wp/pull/6893/files#diff-35d7c50e24ff7dd6479c4fdb3ab29ef26b85fee2c9c8c5c7d0f127448d9d36cfR139-R143

Reply:
I didn't find that action to be cleaner. Plus that action is not officially supported by Google as it says on their GitHub page: `This is not an officially supported Google product, and it is not covered by a Google Cloud support contract.`.

The standard `gcloud` command that is officially supported by Google and Google Cloud is much better IMO. Let me know if you feel strongly about it, I'll update.

Reply:
> I didn't find that action to be cleaner.

Not a big deal. I suppose it's more consistent to use the same tool for uploading and removing.

> Plus that action is not officially supported by Google as it says on their GitHub page: `This is not an officially supported Google product, and it is not covered by a Google Cloud support contract.`.

The same applies to the other actions as well including `auth` and `setup-gcloud`.